ubuntudesign/www.ubuntu.com
# Cloud training

## Done
Corrected some incorrect form URLs for the /cloud/training page

## QA
Check the links for the various forms on `/cloud/training` are what they are on the [copy doc](https://docs.google.com/document/d/1gyxy75qXC5Ta08o6Vnm13yK6aeN4qOnsETB2gZ4e0ZY/edit)
